voluptuous/validators.py

```diff
@@ -933,3 +934,49 @@ def _get_precision_scale(self, number):
             raise Invalid(self.msg or 'Value must be a number enclosed with string')
 
         return (len(decimal_num.as_tuple().digits), -(decimal_num.as_tuple().exponent), decimal_num)
+
+
+class SomeOf(object):
+    """Value must pass at least some validations, determined by the given parameter
+
+    The output of each validator is passed as input to the next.
```

Comment:
description will be improved in the next update of this PR (I'm sure there will be so I'm waiting with pushing it 😄 )

Reply:
Haha.. Let's not rely on it.

Reply:
Will push it asap